docker_dev.sh

```diff
@@ -3,7 +3,7 @@ docker build -f alpine.docker -t diginc/pi-hole:alpine .
 docker tag diginc/pi-hole:alpine diginc/pi-hole:latest
 docker build -f debian.docker -t diginc/pi-hole:debian .
 
-IP=$(ip addr show eth0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)
+IP=$(ip route get 8.8.8.8 | awk '{ print $NF; exit }')
```

Comment:
Good point, I didn't think about the deprecation of eth0 naming.  Going forward it is probably going to help more than hurt.  We should just maybe make it override able with a warning for VPN users

```
IP_LOOKUP=$(ip route get 8.8.8.8 | awk '{ print $NF; exit }') # May not work for VPN / tun0
IP=${IP:-$IP_LOOKUP} # if IP variable is set before running this script, it's value is used
```